exercices/part2/practice3.py

```diff
@@ -0,0 +1,9 @@
+
+
+if __name__ == "__main__":
+    """Write a program that prints your favorite quote. It should give credit to the person who said it on the 
+    next line. """
+
+    print("Everyday is a new box boys.You open it, you look at whats inside. "
+          " Its up to you to determine if its a gift or a coffin.")
+    print("-Jax teller")
```

Comment:
It can be user "/n" to print only 27 characters on a line and the others to be on the next line

Reply:
ok, i will remember that for the future and make the change, thank you for the hint